### ISSUE TITLE
[release]: logstash-oss-with-opensearch-output-plugin

### ISSUE THREAD

**Issue by dblock** [opensearch-project/logstash-output-opensearch#230]
### Did you read the on-boarding document

Coming from https://github.com/opensearch-project/logstash-output-opensearch/issues/228#issuecomment-1719180937, please release `opensearchproject/logstash-oss-with-opensearch-output-plugin` docker image.

### What is the name of your component?

opensearchproject/logstash-oss-with-opensearch-output-plugin

### What is the link to your GitHub repo?

https://github.com/opensearch-project/logstash-output-opensearch/

### Targeted release date

Any date possible.

### Where should we publish this component?

Docker Hub

### What type of artifact(s) will be generated for this component?

opensearchproject/logstash-oss-with-opensearch-output-plugin:latest should be pulling the latest logstash-output-opensearch 2.0.2

### Have you completed the required reviews including security reviews, UX reviews?

this is a minor update

### Have you on-boarded automated security scanning for the GitHub repo associated with this component?

yes

**Comment by dlvenable** [comment]
@dblock , This is correct. We are not releasing updates to the Docker image or the tar.gz file. We are releasing updates to `logstash-output-opensearch` as Ruby Gems which are available [here](https://rubygems.org/gems/logstash-output-opensearch/).

**Comment by dblock** [comment]
> @dblock , This is correct. We are not releasing updates to the Docker image or the tar.gz file. We are releasing updates to `logstash-output-opensearch` as Ruby Gems which are available [here](https://rubygems.org/gems/logstash-output-opensearch/).

So is the docker image EOLed?

**Comment by dlvenable** [comment]
Yes, the Docker image and the tarball are EOL.

**Comment by SebastianThorn** [comment]
> Yes, the Docker image and the tarball are EOL.

Who can add this information to the description for the public docker-image?
It seems to has 30k downloads a week. https://hub.docker.com/r/opensearchproject/logstash-oss-with-opensearch-output-plugin

**Comment by dblock** [comment]
@SebastianThorn We should. @peterzhuamazon do you know how?

**Comment by bbarani** [comment]
@dblock We can add the below information to the description. Let me know if it looks ok.

**DEPRECATION NOTICE:**
This image is no longer supported / maintained (The image was last updated August 10, 2023). Please read this [blog](https://opensearch.org/blog/Discontinuing-custom-Logstash-distribution/) for more information.

**Comment by dblock** [comment]
Looks good to me! Thanks.

**Comment by bbarani** [comment]
> Looks good to me! Thanks.

done!
